res/scripts/newApproach.js

```diff
@@ -0,0 +1,22 @@
+document.addEventListener("DOMContentLoaded", function(event) {
+	onload_image();
+
+  //The array has been created so its available in the memory during the game
+//  for (var faceCard=0; faceCard<=51; faceCard++){
+ //   faceCardsArray[faceCard] = new Image(); //Every element of the array is defined as a new image
+   // faceCardsArray[faceCard].src = "./res/images/face/" + faceCard + ".png"; //every element source has been set
+
+  //}
+ 
+});
+
+
+function onload_image() {
```

Comment:
Puh hard to read PR :) so the new Approach is the correct one? I would consider remove the old approach and never call something new. 
Additionally you do not have to save "old" stuff, as soon as you have it in Git there exists always a revision of it.

Reply:
Sorry this was a sandboxy file... renamed it to donot-review